deployments/a11y/image/environment.yml

```diff
@@ -26,7 +26,7 @@ dependencies:
   #- -r infra-requirements.txt
   # Upgrade separate from what everyone else uses for now
   # https://github.com/berkeley-dsep-infra/datahub/issues/3693
-  - notebook==7.0.0b4
+  - notebook==7.0.0rc1
   - jupyterlab==4.0.1
```

Comment:
Also you should be able to install both `notebook` and `jupyterlab` from conda forge.

Reply:
For installing `notebook` this would mean adding the `conda-forge/label/notebook_rc` channel.